github-dark.css

```diff
@@ -2371,8 +2371,8 @@
   .render-shell img:not(.math) {
     background-image: url("data:image/gif;base64,R0lGODlhCgAKAIAAABoaGgAAACwAAAAACgAKAAACEYQdmYcaDNxjEspKndVZbc8UADs=") !important;
   }
-  .render-shell:hover .img-view img, .file:hover .image img, .file:hover .image canvas, img.asset:hover,
-  .render-shell:hover img:not(.math) {
+  .render-shell:hover .img-view img, .file:hover .image img,
+  .file:hover .image canvas, img.asset:hover, .render-shell:hover img:not(.math) {
```

Comment:
Please add all new selectors next to the `{`, instead of before other (older) selectors.